lib/commands/connect/mapping-reload.js

```diff
@@ -17,16 +17,14 @@ module.exports = {
   needsApp: true,
   needsAuth: true,
   run: cli.command(co.wrap(function* (context, heroku) {
-    let connection = yield api.withConnection(context, heroku);
-    let mapping = yield api.withMapping(connection, context.args.mapping);
+    yield cli.action('reloading mapping', co(function* () {
```

Comment:
We don't have the actual mapping name yet when this line executes, but we do have the name the user passed in. I'll update to use that.